code/game/objects/items/holy_weapons.dm

```diff
@@ -407,7 +407,7 @@
 	AddComponent(/datum/component/butchering, 70, 110) //the harvest gives a high bonus chance
 
 /obj/item/nullrod/scythe/vibro
-	name = "high frequency blade"
+	name = "high fronquency bladd"
```

Comment:
What

Reply:
They probably changed the name so it would not be confused with the real thing and get a chap lynched

Reply:
spot on